spec/controllers/github_users_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe GithubUsersController, type: :controller do
+  let(:user) { create(:github_user, login: "user_login") }
+  let(:platanus_org) { create(:organization, login: "platanus", members: [user]) }
+  let(:github_session) { instance_double("GithubSession") }
+
+  before do
+    allow(github_session).to receive(:token).and_return("token")
+    allow(github_session).to receive(:organizations).and_return(
+      [{ id: platanus_org.gh_id, role: "member" }]
+    )
+    allow(github_session).to receive(:user).and_return(user)
+    allow(subject).to receive(:github_session).and_return(github_session)
+    expect(subject).to receive(:authenticate_github_user).and_return(true)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/NamedSubject: Name your test subject if you need to reference it explicitly.